src/module/Ui/templates/entity/view/partials/input-challenge.twig

```diff
@@ -0,0 +1,21 @@
+{% set revision = entity.getCurrentRevision() %}
```

Comment:
not sure yet. Maybe we need different templates for different types of input challenges. Will review this when we finished the front end.